agent_application/src/metadata/info.rs

```diff
@@ -0,0 +1,50 @@
+use axum::{extract::State, Json};
+use chrono::TimeDelta;
+use serde::Serialize;
+use serde_with::skip_serializing_none;
+
+use crate::metadata::MetadataState;
+
+include!(concat!(env!("OUT_DIR"), "/metadata.rs"));
+
+#[skip_serializing_none]
+#[derive(Serialize)]
+pub struct Info {
+    /// The version of the application.
+    version: Option<String>,
+    /// The git commit hash from which the application was built.
+    git_commit_hash: Option<String>,
+    /// The release channel of the application. Possible values are: `stable`, `next`, `beta`, `canary`.
+    release_channel: Option<String>, // TODO: mapping ok? stable/latest (main), next, beta, canary (alpha)
+    /// The timestamp when the Docker image was built.
+    docker_build_timestamp: Option<String>,
+    /// The current uptime of the application in a human-friendly format.
+    uptime: String,
+}
```

Comment:
Nit:

You could choose to reuse your `Version` struct here so that potential future changes will be automatically be reflected both for the `/version` endpoint as well as here in the `/info` endpoint (if that's desirable):
```rust
#[skip_serializing_none]
#[derive(Serialize)]
pub struct Info {
    #[serde(flatten)]
    version: super::version::Version,
    /// The release channel of the application. Possible values are: `stable`, `next`, `beta`, `canary`.
    release_channel: Option<String>, // TODO: mapping ok? stable/latest (main), next, beta, canary (alpha)
    /// The timestamp when the Docker image was built.
    docker_build_timestamp: Option<String>,
    /// The current uptime of the application in a human-friendly format.
    uptime: String,
}
```